lib/inspec/schema/exec_json.rb

```diff
@@ -1,66 +1,63 @@
 require "lib/inspec/schema/primitives"
 
-# We disable this for the sake of legibility
-# rubocop:disable Layout/AlignHash
-
 # These type occur only when running "inspec exec --reporter json <file>".
 
 module Schema
   module ExecJson
     # Represents a label and description, to provide human-readable info about a control
     CONTROL_DESCRIPTION = SchemaType.new("Control Description", {
       "type" => "object",
-      "additionalProperties"  => false,
-      "required"              => %w{label data},
-      "properties"            => {
+      "additionalProperties" => false,
+      "required" => %w{label data},
+      "properties" => {
         "label" => STRING,
-        "data"  => STRING,
+        "data" => STRING,
       },
     }, [])
 
     # Lists the potential values for a control result
     CONTROL_RESULT_STATUS = SchemaType.new("Control Result Status", {
       "type" => "string",
-      "enum" => %w{passed failed skipped error}
+      "enum" => %w{passed failed skipped error},
     }, [])
 
     # Represents the statistics/result of a control"s execution
     # TODO: Provide descriptions for these types
     CONTROL_RESULT = SchemaType.new("Control Result", {
       "type" => "object",
-      "additionalProperties"  => false,
-      "required"              => %w{status code_desc run_time start_time},
-      "properties"            => {
-        "status"        => CONTROL_RESULT_STATUS.ref,
-        "code_desc"     => STRING,
-        "run_time"      => NUMBER,
-        "start_time"    => STRING,
-  
+      "additionalProperties" => false,
+      "required" => %w{status code_desc run_time start_time},
+      "properties" => {
+        "status" => CONTROL_RESULT_STATUS.ref,
+        "code_desc" => STRING,
+        "run_time" => NUMBER,
+        "start_time" => STRING,
+
         # All optional
-        "resource"      => STRING,
-        "message"       => STRING,
-        "skip_message"  => STRING,
-        "exception"     => STRING,
-        "backtrace"     => OBJECT,
+        "resource" => STRING,
+        "message" => STRING,
+        "skip_message" => STRING,
+        "exception" => STRING,
+        "backtrace" => OBJECT,
```

Comment:
Do we really have a whitespace linter rule turned on? @miah?

I will fight chefstyle on this one. Visual pattern matching is a good thing to the point that we're literally evolved for it. We shouldn't make it harder.